docs/admin/installation/hardware.rst

```diff
@@ -428,7 +428,8 @@ Application and Monitor Servers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 We currently recommend 10-, 11-, and 12-series ASUS (formerly Intel) NUCs for
-SecureDrop servers.
+SecureDrop servers. 10-12th generation NUCs are produced by Intel, after which ASUS
```

Comment:
10-12th is a bit awkward, I think "10th through 12th generation..." would read better?

Reply:
I reworded that entire intro paragraph a bit.